api/resources_portal/middleware/oauth.py

```diff
@@ -0,0 +1,97 @@
+from django.contrib.auth import login
+
+import furl
+import orcid
+import requests
+
+from resources_portal.models.grant import Grant
+from resources_portal.models.organization import Organization
+from resources_portal.models.user import User
+
+CLIENT_ID = "APP-2AHZAK2XCFGHRJFM"
+CLIENT_SECRET = "1aaa0a94-5605-4cd8-95a4-6724c2f7fd42"
```

Comment:
Oh dear, I didn't realize this repo was public. David recommended that we use this compromised client secret for testing, then ask for a refresh once the infrastructure for storing it (like git crypt) is set up.

Reply:
We already have the infrastructure for storing secrets: https://github.com/AlexsLemonade/resources-portal/settings/secrets

Basically what you want to do is read it in as an environment variable, then when the system is deployed we'll provide the real value from Github's secrets.

The one question that leaves is how it will work in development though?

Also just to note: it is definitely not best practices to include secrets in plaintext even if it were a private repo.